core/chains/evm/txmgr/eth_confirmer.go

```diff
@@ -1056,7 +1056,7 @@ func (ec *EthConfirmer) markForRebroadcast(etx EthTx, head txmgrtypes.Head) erro
 // This operates completely orthogonal to the normal EthConfirmer and can result in untracked attempts!
 // Only for emergency usage.
 // This is in case of some unforeseen scenario where the node is refusing to release the lock. KISS.
-func (ec *EthConfirmer) ForceRebroadcast(beginningNonce uint, endingNonce uint, gasPriceWei uint64, address gethCommon.Address, overrideGasLimit uint32) error {
+func (ec *EthConfirmer) ForceRebroadcast(beginningNonce uint64, endingNonce uint64, gasPriceWei uint64, address gethCommon.Address, overrideGasLimit uint32) error {
```

Comment:
Changed to uint64 because in the database the nonce stored as a BigInt (8 bytes / 64 bits) anyway as well as other methods. I think it was a uint only because the cli also accepted a uint for the ForceRebroadcast argument.

Reply:
Sounds good for EVM at least https://eips.ethereum.org/EIPS/eip-2681

Reply:
In database its an int64, actually there are some uint64 values that will not serialize to this so I usually opt to use `int64` instead. But this is a rarely used function so it's not that important.